Plugins/GeoModel/include/Acts/Plugins/GeoModel/GeoModelBlueprintCreater.hpp

```diff
@@ -0,0 +1,160 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2024 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Detector/Blueprint.hpp"
+#include "Acts/Detector/KdtSurfacesProvider.hpp"
+#include "Acts/Detector/interface/IInternalStructureBuilder.hpp"
+#include "Acts/Geometry/Extent.hpp"
+#include "Acts/Geometry/GeometryContext.hpp"
+#include "Acts/Geometry/VolumeBounds.hpp"
+#include "Acts/Utilities/BinningData.hpp"
+#include "Acts/Utilities/Logger.hpp"
+
+#include <map>
+#include <memory>
+#include <optional>
+#include <string>
+#include <tuple>
+#include <vector>
+
+namespace Acts {
+
+struct GeoModelTree;
+
+class GeoModelBlueprintCreater {
```

Comment:
I missed this, but what's the plan with the explicit use of SQL here? My latest info was that we'd use GM's high-level API to talk to the geometry.

Reply:
Yeah, it's not yet entirely clear if GeoModel can have all the information, that's currently a workaround to see what sort of info we need to build a complete Gen2 geometry.

I had talked to Joe Boudreaux already, and I think we wanted to revisit that once we have a prototype.